src/common/utils.ts

```diff
@@ -911,3 +911,14 @@ const replaceLikeOperatorRecursive = (
 export const sleep = (ms: number) => {
   return new Promise((resolve) => setTimeout(resolve, ms));
 };
+
+/**
+ * Accepts either an object or array of objects.
+ *
+ * Note that T should not extend Array (eg no nested arrays)
+ * @param x object or array of objects
+ * @returns array containing x
+ */
+export function oneOrMore<T>(x: T[] | T): T[] {
```

Comment:
Is it a bad idea to allow the configuration to have either a single object or an array of objects? I'm not sure whether this is convenient or confusing.

Reply:
Generally speaking, it might be confusing for the user and a simpler solution might be available. 
That said, it might be that it is needed in some cases.

We should check if we use it anywhere in the code

Reply:
On a second thought, I would define oneOrMore as an array of one or more items of the same type, but an array nevertheless.